Integrations/integration-EWSv2.yml

```diff
@@ -143,6 +143,7 @@ script:
     LAST_RUN_TIME = "lastRunTime"
     LAST_RUN_IDS = "ids"
     LAST_RUN_FOLDER = "folderName"
+    ERROR_COUNTER = "errors"
```

Comment:
@BenJoParadise wasn't it different naming in the other PR? please merge from master to this one and resolve conflicts

Reply:
Merged